concepts/enums/about.md

```diff
@@ -1 +1,136 @@
 # Enums
+
+Enums are cairo data type used to define a set of predefined variants for the purpose of code readability and safety. In Cairo, `enum` variants can be declared to hold different data types (uint, struct, tuples, arrays, dictionaries, core default library, other enums).
+
+The code snippet below shows how enums could be defined:
+
+### Simple example of an enum with no associated value
+
+```Rust
+    #[derive(Drop)]
+    enum Direction {
+        North,
+        East,
+        South,
+        West,
+    }
+```
+
+The `Direction` enum declared above is a simple enum with four variants: `North`, `East`, `South`, and `West`. Notice that the variants are declared in PascalCase, being the recommended naming convention. Each variant represents a distinct value of the `Direction` enum type. Each variant in this particular enum example has no associated value and be instantiated using this syntax:
+
+```Rust
+    let direction = Direction::North;
+```
+
+### Simple example of an enum with associated value
+
+```Rust
+    #[derive(Drop)]
+    enum Direction {
+        North: u128,
+        East: u128,
+        South: u128,
+        West: u128,
+    }
+```
+
+It is evident from the second example that each variant in the `Direction` enum has an associated value of `u128`. This kind of enum is instantiated as follows:
+
+```Rust
+    let direction = Direction::North(10);
+```
+
+### Complex example of an enum with combined custom types
+
+```Rust
+    #[derive(Drop, Serde, Copy, starknet::Store)]
```

Comment:
Remove all the unnecessary derived traits from all enums

Reply:
In this case, starknet::Store is the only unnecessary trait?